bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/container.jsx

```diff
@@ -43,6 +42,7 @@ const UserOptionsContainer = withTracker((props) => {
       intl.formatMessage(intlMessages.clearStatusMessage), 'info', 'clear_status',
     );
   };
+  const currentUser = Users.findOne({ userId: Auth.userID });
```

Comment:
Another unprotected find that will cause it all to re-render when irrelevant properties are changed.